openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/courseware.html

```diff
@@ -1,27 +1,65 @@
 {% extends 'openedx_lti_tool_plugin/base.html' %}
+{% load i18n %}
+{% load list_utilities %}
 
-{% block title %}LTI Courseware{% endblock %}
-
+{% block title %}{{ unit_obj.display_name }} | {% translate 'LTI Courseware' %}{% endblock %}
 {% block body %}
-<iframe
-  id='unit-iframe'
-  src='{{ xblock_url }}'
-  style='width: 100%;border: none;display: block;'
-  scrolling='no'>
-</iframe>
-
+<div class='container-fluid container-lg my-4 my-xl-5'>
+  <main>
+    <button type='button' class='btn btn-secondary' data-bs-toggle='offcanvas' data-bs-target='#courseSidebar' aria-controls='courseSidebar'>
+      <i class='fa fa-bars' aria-hidden='true'></i>
+      {% translate 'Course Outline' %}
+    </button>
+    <iframe id='unit-iframe' class='w-100 border-0 d-block' src='{% url request.resolver_match.app_name|add:":lti-xblock" unit_obj.id%}' scrolling='no'></iframe>
+    <div class='d-flex gap-2 justify-content-center align-items-center'>
+      {% with unit_index=units|get_element_index:unit_obj.id %}
+        {% if unit_obj.id != units|first %}
+          {% with previous_unit_index=unit_index|add:'-1' %}
+            <a href='{% url request.resolver_match.app_name|add:":lti-courseware" course_id units|get_by_index:previous_unit_index %}' class='btn btn-outline-secondary px-4'>
+              <i class='fa fa-chevron-left' aria-hidden='true'></i> {% translate 'Previous' %}
+            </a>
+          {% endwith %}
+        {% else %}
+          <a href='#' class='btn btn-outline-secondary px-4 disabled'>
+            <i class='fa fa-chevron-left' aria-hidden='true'></i> {% translate 'Previous' %}
+          </a>
+        {% endif %}
+        {% if unit_obj.id != units|last %}
+          {% with next_unit_index=unit_index|add:'1' %}
+            <a href='{% url request.resolver_match.app_name|add:":lti-courseware" course_id units|get_by_index:next_unit_index %}' class='btn btn-outline-secondary px-4'>
+              {% translate 'Next' %} <i class='fa fa-chevron-right' aria-hidden='true'></i>
+            </a>
+          {% endwith %}
+        {% else %}
+          <a href='#' class='btn btn-outline-secondary px-4 disabled'>
+            {% translate 'Next' %} <i class='fa fa-chevron-right' aria-hidden='true'></i>
+          </a>
+        {% endif %}
+      {% endwith %}
+    </div>
```

Comment:
@alexjmpb What do you think about moving the next and previous navigation buttons to a separate template?